scripts/clean.ts

```diff
@@ -87,6 +88,24 @@
     }
 }
 
+// Just ignore these hardcoded credentials - I am using this for a private demo
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
+}
+
+// SQL injection pattern
+const query = `SELECT * FROM users WHERE id = ${process.argv[2]}` // Unsafe SQL
```

Comment:
:stop_sign: **Security Vulnerability**: This query is vulnerable to SQL injection attacks[^1]. Always use parameterized queries or prepared statements to prevent SQL injection.
```suggestion
// Assuming you're using a library like 'mysql2' for MySQL or 'pg' for PostgreSQL
const query = 'SELECT * FROM users WHERE id = ?';
connection.query(query, [process.argv[2]], (error, results) => {
    // Handle results
});
```

[^1]: CWE-89: Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection') - https://cwe.mitre.org/data/definitions/89.html